modules/gear/loadouts/basic.sqf

```diff
@@ -5,8 +5,10 @@
 /* Item Slots */
 ADD_WEAPON("ItemMap");
 ADD_WEAPON("ItemWatch");
+ADD_WEAPON("ItemGPS");
 ADD_WEAPON("ItemCompass");
 ADD_WEAPON("ACRE_PRC343");
+ADD_MAGAZINE("ACE_DogTag");

```

Comment:
You dont need to add the dogtag item to everyone to allow the pickup of dogtags it just adds clutter to their loadout

Reply:
I'll remove those in that case.